src/functions/normTV.jl

```diff
@@ -0,0 +1,111 @@
+# Total variation norm (times a constant)
+
+export NormTV
+
+"""
+**``TV`` norm**
```

Comment:
Maybe you could mention “1-dimensional” here so that it’s immediately clear.

Reply:
done